public/app/features/alerting/unified/components/rules/central-state-history/CentralAlertHistoryScene.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+import { getDataSourceSrv } from '@grafana/runtime';
+import {
+  EmbeddedScene,
+  PanelBuilders,
+  SceneControlsSpacer,
+  SceneFlexItem,
+  SceneFlexLayout,
+  SceneQueryRunner,
+  SceneReactObject,
+  SceneRefreshPicker,
+  SceneTimePicker,
+} from '@grafana/scenes';
+import {
+  GraphDrawStyle,
+  GraphGradientMode,
+  LegendDisplayMode,
+  LineInterpolation,
+  ScaleDistribution,
+  StackingMode,
+  TooltipDisplayMode,
+  VisibilityMode,
+} from '@grafana/schema/dist/esm/index';
+
+import { DataSourceInformation, PANEL_STYLES } from '../../../home/Insights';
+import { SectionSubheader } from '../../../insights/SectionSubheader';
+
+import { HistoryEventsListObjectRenderer } from './CentralAlertHistory';
+
+export const CentralAlertHistoryScene = () => {
+  const dataSourceSrv = getDataSourceSrv();
+  const ashDs: DataSourceInformation = {
+    type: 'loki',
+    uid: 'grafanacloud-alert-state-history',
```

Comment:
(Just noting the things that didn't work locally for me without changes, as per Slack)
I had to rename this to match my already-set-up-Loki (`gdev-loki`)

Reply:
I added some notes in the PR description with the steps to run this PR locally